src/main/java/lessons/hw2/ListNode.java

```diff
@@ -0,0 +1,60 @@
+package lessons.hw2;
+
+public class ListNode {
+    int val;
+    ListNode next = null;
+
+    ListNode() {}
+    ListNode(int val) {
+        this.val = val;
+    }
+    ListNode(int val, ListNode next) {
+        this.val = val;
+        this.next = next;
+    }
+
+    public static ListNode reverseList(ListNode head) {
+        if(head == null) {
```

Comment:
```suggestion
        if (head == null) {
```
Please configure checkstyle

Reply:
ok